server/models/blogs.js

```diff
@@ -0,0 +1,34 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const BlogsSchema = new Schema({
+  userId: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  content: [
+    {
+      _id: Schema.Types.ObjectId,
+      type: Schema.Types.Mixed,
+      content: {
+        type: Schema.Types.Mixed,
+        required: true,
+      },
+    },
```

Comment:
`Content` has a child as `content` which seems confusing.

Reply:
Also, I think this is not the appropriate use of Mixed type schema

Reply:
Both issues addressed and resolved